samples/angular-todo/src/app/views/login/login.component.html

```diff
@@ -11,18 +11,25 @@
 <div class="cstm_container_v-centered container-fluid d-flex align-items-center">
   <div class="w-100">
     <app-back-home></app-back-home>
+    <ng-container *ngIf="!isWebAuthn; else webAuthn">
```

Comment:
A few things here:

1) Because I'm not super familiar with Angular, this isn't immediately obvious what's going on to me, but the model is hardcoding `this.isWebAuthn` to `false`, so the condition here of `!isWebAuthn` will always evaluate to `true` on the first pass. Yeah?

Then, you bind a function called `onSetIsWebAuthn` to an event named `isWebAuthnn` (is the double "n" necessary?). When this event is emitted, you're flipping the `this.isWebAuthn` to true? This makes the WebAuthn component to render. Is that correct?

2) Either way, I don't believe we should the logic for WebAuthn step at this level. The login view is a bit too specific to make these decisions. We should be making this decision a bit deeper in the component tree, like in `form.component.ts`. I say this because WebAuthn can be used in both login AND registration. As it is right now, WebAuthn couldn't be used for any registration flow.

What's more, I think this logic would prevent WebAuthn from being the first node in a tree, which that too is possible. Let me know if you have any questions about this.

Reply:
1. the first assumpiton is correct. Depending on the emitted value login component will be loading or not the webAuthn component. 
2. it'd take some changes as i made all of the implementation based on the premise that it's either the webAuthn funcionality or the form component as this comment says  "Since WebAuthn is being handled as its own unique component, it is necessary to check if we have WebAuthn type step. if so an output event is emitted to the log-in component to load the webAuhtn component."
Before startibng implementing, after a talk i had to ryan we decided to go this way, and of course i accepted as a valid option. I'd like to discuss it the three of us if possible.

Reply:
Hey Juan, sorry for the confusion. Please go ahead and push your changes, so we can see your solution. Remember, as long as the last commit in your Git history is **_your_** commit, you can use `git commit --amend` and include your changes in that existing commit. That way, you don't have to add additional commits when addressing PR comments. 

The only thing is when you amend a commit (or rebase), it rewrites history, so when you push, you have to force push. This is fine as long as you're only rewriting your own commit, and you're force pushing to a branch that is not shared with anyone (like if you're pair programming with someone and sharing a branch). To force push, you'd use `git push -f origin webAuthn`. 

If, for any reason, you'd like to know why force pushing is necessary, or what rewriting history means, please read this article I wrote. Others find it quite helpful: https://cerebralideas.com/blog/rewriting-git-history

Reply:
thanks i will ready it! commiting my changes soon

Reply:
done